<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>338</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gBHantek">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>HANTEK</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QGroupBox" name="gBHorizontal">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>160</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Horizontal</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="1" rowspan="2">
          <widget class="QDial" name="dialTimDiv">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>60</height>
            </size>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="wrapping">
            <bool>false</bool>
           </property>
           <property name="notchesVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <layout class="QVBoxLayout" name="vLTDiv">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Buffer length</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cBBufLen">
             <item>
              <property name="text">
               <string>1 024 (1K)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2 048 (2K)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3 072 (3K)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4 096 (4K)</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Format</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cBFormat">
             <item>
              <property name="text">
               <string>Y - T</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>X - Y</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Time/DIV</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cBTimDiv">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>2nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>200nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500nS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>200uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500uS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>200mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500mS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>10S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>20S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>50S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>100S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>200S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>500S</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1000S</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="gBTrigger">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Trigger</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gLTrigger">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="1" column="1">
            <widget class="QComboBox" name="cBTrigSweep">
             <item>
              <property name="text">
               <string>Auto</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Normal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Single</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="cBPulsePolar">
             <item>
              <property name="text">
               <string>+</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>-</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cBTrigSrc"/>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cBTrigMode">
             <item>
              <property name="text">
               <string>Edge</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pulse</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Video</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CAN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>LIN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UART</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SPI</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>IIC</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ALT</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Trigger mode</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Trigger sweep</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="lPulseWid">
             <property name="text">
              <string>Pulse width</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Pulse polarity</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="cBPwCond">
             <item>
              <property name="text">
               <string>Equals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Not equals</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Above</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Below</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="lPWCond">
             <property name="text">
              <string>PW condition</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Trigger source</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lEPulseWid">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cBPulseUnit">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <bold>false</bold>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>ns</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>us</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ms</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>s</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cBTimDiv</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>dialTimDiv</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialTimDiv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cBTimDiv</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
